YOLayout/YOLayout.m

```diff
@@ -65,7 +65,17 @@ - (CGSize)sizeThatFits:(CGSize)size {
 }
 
 - (CGRect)setFrame:(CGRect)frame view:(id)view sizeToFit:(BOOL)sizeToFit {
-  return [self setFrame:frame view:view options:(sizeToFit ? YOLayoutOptionsSizeToFit : 0)];
+
+  YOLayoutOptions options = 0;
+  // If UILabel and 1 line, we need to fix size to fit options to constrain width,
+  // since UILabel will return a size > the width we specify.
+  if (sizeToFit && [view isKindOfClass:UILabel.class] && ((UILabel *)view).numberOfLines == 1) {
```

Comment:
In fact we could make

YOLayoutOptionsSizeToFit | YOLayoutOptionsConstrainWidth the default FOR ALL views when sizeToFit:YES is specified?

You can always use options to override defaults.

Do you ever actually want your subview to be wider than the parent view? (I mean maybe once in a blue moon, but otherwise its usually a bug.)